llama_stack/providers/inline/post_training/torchtune/common/utils.py

```diff
@@ -56,6 +70,13 @@ class DatasetSchema(BaseModel):
     ),
 }
 
+DATA_FORMATS: Dict[str, Transform] = {
+    "alpaca": AlpacaToMessages,
+    "instruct": InputOutputToMessages,
+    "chat_sharegpt": ShareGPTToMessages,
+    "chat_openai": OpenAIToMessages,
+}
+
 
 EXPECTED_DATASET_SCHEMA = DatasetSchema(
```

Comment:
Might be helpful to use DataSchemaValidatorMixin for data schema validation https://github.com/meta-llama/llama-stack/blob/main/llama_stack/providers/utils/common/data_schema_validator.py#L65

Reply:
Synced offline: we can keep logic for validation in common utils, but there's no value in having DataSchemaValidatorMixin. Refactor in https://github.com/meta-llama/llama-stack/pull/720